cmd/gouroboros/main.go

```diff
@@ -35,8 +35,14 @@ type globalFlags struct {
 }
 
 func newGlobalFlags() *globalFlags {
+	var name string
+	if os.Args == nil {
+		name = "gouorobos"
```

Comment:
`gouroboros`

Reply:
It's not that I misspelled it. It's that I did it twice.